doc/sphinx/changes.rst

```diff
@@ -64,6 +64,22 @@ Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
 contributions from many users. A list of packages is available at
 https://coq.inria.fr/opam/www/.
 
+During the 8.12 development cycle, there was a switch in the
+maintenance process (see `#11295
+<https://github.com/coq/coq/pull/11295>`_) to adopt the idea that
+every component of the system should have a dedicated team of
+maintainers (in charge of reviewing and merging incoming pull
+requests).  This provided an opportunity to significantly expand the
+list of maintainters.  Special thanks to them!  Our current 31
```

Comment:
```suggestion
Prior to 8.12, the code maintainers for each component of |Coq| were defined
directly in the file `CODEOWNERS` (such as @Zimmi48), which required a pull request to update it.
In 8.12,  `#11295<https://github.com/coq/coq/pull/11295>`_ now defines code maintainers
with GitHub teams (such as @coq/doc-maintainers) that can be updated more easily and
are visible in GitHub tooltips.  This provided an opportunity to significantly expand the
list of maintainters.  Special thanks to them!  Our current 31
```

> to adopt the idea that
every component of the system should have a dedicated team of
maintainers (in charge of reviewing and merging incoming pull
requests).

This part isn't new, is it?

Reply:
This change goes beyond just the easiness of adding people to the maintainer teams: before this change each component (except those that had already adopted teams) had a *single* principal maintainer and a *single* secondary maintainer. This meant that most often, if principal maintainers were not available or responsive, PRs could get ignored. The maintainer teams idea puts team members on equal footing and ensures that more people will be made aware of incoming PRs (and thus they are less likely to get ignored). It also introduces the culture of expanding the teams as soon as you have a competent contributor who is relevant to maintain a specific component, instead of looking for new maintainers only when a component is badly maintained because its current principal maintainer doesn't have enough time for this anymore.

I'm not against slightly tweaking the phrasing, but it should remain short and to the point, rather than entering into technical details.

Reply:
I wasn't part of the conversation.  I thought doc-maintainers already existed in some form prior to the PR but maybe I'm mistaken.

Seems to me the key point is changing from a single designated maintainer to a team of maintainers.

"to adopt the idea that every component of the system should have a dedicated team of
maintainers (in charge of reviewing and merging incoming pull requests)" doesn't seem very new.

Reply:
doc-maintainers already existed in the current form prior to the PR. This switch was just the decision to adopt such teams for *every* component.

>Seems to me the key point is changing from a single designated maintainer to a team of maintainers.

Yes!

>"to adopt the idea that every component of the system should have a dedicated team of
maintainers (in charge of reviewing and merging incoming pull requests)" doesn't seem very new.

OK, I'll try to tweak the wording.